src/msw/toplevel.cpp

```diff
@@ -306,6 +337,18 @@ WXLRESULT wxTopLevelWindowMSW::MSWWindowProc(WXUINT message, WXWPARAM wParam, WX
 #endif // #ifndef __WXUNIVERSAL__
             }
             break;
+
+        case WM_DPICHANGED:
+            {
+                const int xDPI = (int)LOWORD(wParam);
```

Comment:
Are the casts to int really necessary? I fail to see what do they add.

I'd probably just write `wxSize(LOWORD(wParam), HIWORD(wParam))` below directly anyhow...